src/qonnx/core/modelwrapper.py

```diff
@@ -542,6 +542,15 @@ def get_node_index(self, node):
         except ValueError:
             return None
 
+    def get_node_from_name(self, node_name):
+        """Returns the node with the specified name."""
+        try:
+            for node in self.graph.node:
+                if node.name == node_name:
+                    return node
+        except ValueError:
+            return None
+
```

Comment:
note: I see this pattern here is taken from the `get_node_index` function above. when no match is found (either for the node_name or for finding the index, for the function above) these functions rely on the Python behavior that returns an implicit `None` when no return statement is provided: https://realpython.com/python-return-statement/#:~:text=That%20default%20return%20value%20will%20always%20be%20None%20.&text=If%20you%20don't%20supply,None%20as%20a%20return%20value.

it would be better to explicitly return None for these cases, I will make this change.